MapboxMobileEvents/MMETimerManager.m

```diff
@@ -19,6 +19,8 @@ - (instancetype)initWithTimeInterval:(NSTimeInterval)timeInterval target:(id)aTa
 
 - (void)start {
     [self cancel];
+    // NOTE: this has the side-effect of scheduling the timer on the current or main run loop.
+    // Once delayed startup is implemented, the scheduling of this timer may also need to be delayed
```

Comment:
Is there a plan to test this pattern in the field, based on this comment? We need to be careful about the memory management implications of using `NSTimer`. I like that it's encapsulated in a manager class which should make it easier to break any potential memory retain cycles, but nonetheless it's something we're going to want to put some additional thought into.

Reply:
I think we could remove this comment. The delayed startup has been field tested to a fair extent at this point. Agreed on points of `NSTimer` - certainly something to keep an eye on. 👁